cabot_ui/scripts/cabot_ui_manager.py

```diff
@@ -468,9 +468,11 @@ def description_callback(result):
             try:
                 if result:
                     self._logger.info(f"description - {result=}")
+                    self._interface.publish_navigation_event_sound("ImageDescResponseReceived")
```

Comment:
Send the event in the describe_surround/describe_error/etc method instead of sending it separately.
Method calls of the interface should be as abstracted as possible, not coupled with the actual actions (such as speak, publish topic), so that the interface can be replaceable with a different implementation of the interface (such as a test).